otx/algorithms/common/tasks/training_base.py

```diff
@@ -108,6 +108,7 @@ def _run_task(self, stage_module, mode=None, dataset=None, **kwargs):
         if dataset is not None:
             train_data_cfg = Stage.get_train_data_cfg(self._data_cfg)
             train_data_cfg["data_classes"] = data_classes
+            train_data_cfg["classes"] = data_classes
```

Comment:
Why we need `train_data_cfg["classes"]` ? can't we use existing one(`train_data_cfg["data_classes"]`)?

Reply:
When I modified the data loading part to be able to load images from a JSON file for the multiclass classification case (https://github.com/openvinotoolkit/training_extensions/pull/1406/commits/cf929c3ab77d3791863337e34494a4012f1e8b97), somehow, it complained that it wanted to see the classes using the `classes`, so I just added that.

I need to revert those two changes, since that was only made for me to be able to easily subsample classification datasets without haveing to duplicate the tree structure several times.